### PR TITLE
Added latest release-notes

### PR DESCRIPTION
**Pull request by StefanKert**

### CODE REVIEW

**Review by fiscoal — APPROVED** [review]

File: portal/portal-sprint-98.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- [Reworked Queue DE SCU connection page (DE)](#reworked-queue-de-scu-connection-page-de)
+
+### User Management
+- [Reworked Outlets page](#reworked-outlets-page)
+- [Added email verification field to registration dialog](#added-email-verification-field-to-registration-dialog)
+
+### Tools
+- [Introduce tool for generating AO148 applications (DE)](#introduce-tool-for-generating-ao148-applications-de)
+ 
+## Features
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-98.md:26]: Duplicated "Feature" Header

File: portal/portal-sprint-98.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+### Reworked Queue DE page (DE)
+The Queue DE page has been reworked. Even though it appears to be similar to the old page, offering the same functionalities, response times of the page are faster and the overall data overview is improved.
+ 
+### Reworked Queue DE SCU connection page (DE)
+The Queue DE SCU connection page has been reworked. Even though it appears to be similar to the old page, offering the same functionalities, response times of the page are faster and the overall data overview is improved.
+ 
+## User Management
+ 
+### Reworked Outlets page
+The Outlets page has been reworked. Even though it appears to be similar to the old page, offering the same functionalities, response times of the page are faster and the overall data overview is improved.
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-98.md:39]: These three points say exactly the same. We should sum them up. e.g.:
## Improving Page Performance
We invested some time in improving how our tables are displayed. Visually nothing major should have changed but we managed to reduce updates on the tables and reduced necessary memory.
A wider range of monitor resolutions should now also be supported.
Affected by this change are the tables under the following views: Queue, ....

File: portal/portal-sprint-98.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ 
+### Added email verification field to registration dialog
+Very often, customers were accidentally entering a misspelled email address, and were thus unable to login. By adding a confirmation email, the user has to submit it for a second time, so an error will be issued if the two entries don’t match.
+ 
+![VerificationEmail](images/sprint-98/VerificationEmail.png)
+ 
+## Tools
+ 
+### Introduce tool for generating AO148 applications (DE)
+Since many of the PosOperators haven’t had the chance to roll out everything the way the new regulations require, the authorities have given them the opportunity to create the  so-called” AO 148 Antrag” that can be created to extend the period. We wanted to give customers the possibility to create those applications on their own, and also enable them to send it to the authorities very easily.
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-98.md:49]: the so-called” AO
swap " with spacebar

File: portal/portal-sprint-98.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+# fiskaltrust.Portal - Sprint 98
+_April 19, 2021_
+
+**Introduce tool for generating AO148 applications**
+
+In this sprint, we have been focusing on creating a tool that should give users the ability to easily create AO 148 applications.
+
+## Features
+
+### Middleware Configuration
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-98.md:15]: If we sum up the performance improvements we should also change the index.

File: portal/portal-sprint-101.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+slug: /release-notes/portal/sprint-101
+title: Sprint 101 (May 31, 2021)
+---
+
+# fiskaltrust.Portal - Sprint 101
+_May 31, 2021_
+
+**Usability improvements**
+
+In this sprint, we have introduced several usability improvements including improving the overall performance.
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-101.md:11]: "several" implies more than two improvements :)

File: portal/portal-sprint-101.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+## Features
+
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-101.md:14]: Should we also mention this Pbi in the release notes?
#42260 Integrate basic read tracking into KBA visualization

File: portal/portal-sprint-101.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+### Middleware Configuration
+- [Visual improvements to Middleware component dialogs](#visual-improvements-to-middleware-component-dialogs)
+- [Feedback on Cashbox rebuild](#feedback-on-cashbox-rebuild)
+
+## Middleware Configuration
+
+### Visual improvements to Middleware component dialogues
+
+Some minor visual improvements of the TableViews in Portal (DE) have been implemented. We invested some time in improving the way our tables are displayed. Visually nothing major should have changed, aside from a smaller screen size -in which the user is navigating the TableViews page- does not affect anymore the way the tables are shown. The performance of the page in different resolutions has also been improved.
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-101.md:23]: Those Middleware configuration page updates are actually not DE specific.

Comment (FlorianStadlberger) [reply portal/portal-sprint-101.md:23]: Also small point: Search improved.
As we no longer include the object structure in the Search filter.
Previous:
Searchterm: Nam -> Testing: MyObject**Nam**e: Hubert -> Hubert would show up in the search results
Now:
SearchTerm: Nam -> Testing: Hubert -> no longer added to the results

File: portal/portal-sprint-101.md (hunk 5 above)
Comment (DoxaPap) [reply portal/portal-sprint-101.md:11]: touché :D

**Review by FlorianStadlberger — APPROVED** [review]

**Pull request merged**